orgs/altinn-orgs.json

```diff
@@ -97,6 +97,17 @@
                 "production"
             ]
         },
+        "din": {
+            "name": {
+                "en": "The Norwegian Courts",
+                "nb": "Domstolene i Norge",
+                "nn": "Domstolane i Noreg"
+            },
+            "logo": "https://orglogo.digdir.no/api/logo/org/984195796",
```

Comment:
We should upload this logo to our cdn, but can use this service as the source 😊

Reply:
hehe, little to fast here. let me fix that.